packages/keystone/src/lib/core/mutations/create-update.ts

```diff
@@ -127,6 +103,24 @@ export async function updateOne(
   return updatedItem;
 }
 
+export function updateMany(
+  { data }: { data: { where: Record<string, any>; data: Record<string, any> }[] },
+  list: InitialisedList,
+  context: KeystoneContext,
+  provider: DatabaseProvider
+) {
+  const writeLimit = pLimit(provider === 'sqlite' ? 1 : Infinity);
+  return data.map(async ({ data: rawData, where: rawUniqueWhere }) => {
+    const item = await getAccessControlledItemForUpdate(list, context, rawUniqueWhere, rawData);
+    const { afterChange, data } = await resolveInputForCreateOrUpdate(list, context, rawData, item);
+    const updatedItem = await writeLimit(() =>
+      getPrismaModelForList(context.prisma, list.listKey).update({ where: { id: item.id }, data })
+    );
+    afterChange(updatedItem);
```

Comment:
(i know this isn't a change with this PR)

```suggestion
    await afterChange(updatedItem);
```

Reply:
Oh, nice catch!